Sources/TuistKit/Services/ShareService.swift

```diff
@@ -371,18 +375,30 @@ struct ShareService {
                 displayName: app,
                 version: appBundles.first?.infoPlist.version.description,
                 bundleIdentifier: appBundles.first?.infoPlist.bundleId,
+                icon: appBundles
+                    .concurrentFlatMap { try await iconPaths(for: $0) }
+                    .first,
                 fullHandle: fullHandle,
                 serverURL: serverURL,
                 json: json
             )
         }
     }
 
+    private func iconPaths(for appBundle: AppBundle) async throws -> [AbsolutePath] {
+        try await appBundle.infoPlist.bundleIcons?.primaryIcon?.iconFiles
+            .map { appBundle.path.appending(component: $0 + "@2x.png") }
```

Comment:
This is the convention for iOS files at least. We can update this if needed for other platforms if we find this is not working well.

Reply:
What about leaving this in code for future in-context reference?